lib/MusicBrainz/Server/Form/Work.pm

```diff
@@ -68,6 +68,35 @@ sub is_empty_attribute {
     return !$field->field('type_id')->value && !$field->field('value')->value;
 }
 
+sub validate_languages {
+    my $self = shift;
+
+    my @languages = $self->field('languages')->fields;
+
+    my $is_valid = 1;
+
+    # If we have only one language, then it can be whatever
+    # If we have two or more, we check it's not mul (248) nor zxx (486)
+    # because combining those with anything else doesn't make sense
+    if (scalar @languages > 1) {
+        for my $language_field (@languages) {
+            if ($language_field->value == 284) {
+                $language_field->add_error(
+                    l('You can’t select “~[Multiple languages~]” and specific languages at the same time.')
```

Comment:
Considered translating `[Multiple languages]` (and `[No lyrics]` below) separately as {multiple_languages_name} or whatnot to ensure it uses the same name as the actual language in translation, but I'm actually not sure that is safe because the translation in a sentence might not be the same as the standalone one in some languages.

Reply:
IMHO, using `{lang}` placeholder would probably be safer because these are specific values not a free wording. Maybe @mihkel-t has a different point of view?

Otherwise, what is `~` doing here?

Reply:
I'd say that for translators, it's definitely better to have the full string visible, not a {lang} placeholder. Might be a good idea to have a msgctx saying that "[multiple languages]" should be translated the same way as in the language list.

Reply:
`~` escapes the brackets - I got an error otherwise, which specifically suggested using `~`, so I did!

Reply:
I have not been able to get that error message. You should probably find another way to get around that because this escape character is not currently used for other `[` occurrences. It would be nice to prevent introducing yet another markup syntax if possible.

Reply:
Made this change. If we don't generally want to pass these through FormHandler localization, should we always use push_errors instead of add_error?

Reply:
I'd say yes, ideally, though I'm not sure `add_error` causes any problems unless you clash with the maketext syntax by accident.